webhook.rst

```diff
@@ -0,0 +1,133 @@
+component.
+
+The webhook component routing needs to be defined:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/framework.yaml
+        framework:
+            webhook:
+                routing:
+                    mailer_mailgun:
+                        service: 'mailer.webhook.request_parser.mailgun'
+                        secret: '%env(MAILER_MAILGUN_SECRET)%'
+
+    .. code-block:: xml
+
+        <!-- config/packages/framework.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns:framework="http://symfony.com/schema/dic/symfony"
+                   xsi:schemaLocation="http://symfony.com/schema/dic/services
+                        https://symfony.com/schema/dic/services/services-1.0.xsd
+                        http://symfony.com/schema/dic/symfony https://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+            <framework:config>
+                <framework:webhook enabled="true">
+                    <framework:routing type="mailer_mailgun">
+                        <framework:service>mailer.webhook.request_parser.mailgun</framework:service>
+                        <framework:secret>%env(MAILER_MAILGUN_SECRET)%</framework:secret>
+                    </framework:routing>
+                </framework:webhook>
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/framework.php
+        use App\Webhook\MailerWebhookParser;
+        use Symfony\Config\FrameworkConfig;
+        return static function (FrameworkConfig $frameworkConfig): void {
+            $webhookConfig = $frameworkConfig->webhook();
+            $webhookConfig
+                ->routing('mailer_mailgun')
+                ->service('mailer.webhook.request_parser.mailgun')
+                ->secret('%env(MAILER_MAILGUN_SECRET)%')
+            ;
+        };
+
+Currently, the following third-party mailer services support webhooks:
+
+=============== ==========================================
+Mailer service  Parser service name
+=============== ==========================================
+Mailgun         ``mailer.webhook.request_parser.mailgun``
+Postmark        ``mailer.webhook.request_parser.postmark``
+Sendgrid        ``mailer.webhook.request_parser.sendgrid``
+=============== ==========================================
+
+Set up the webhook in the third-party mailer. For Mailgun, you can do this
+in the control panel. As URL, make sure to use the ``/webhook/mailer_mailgun``
+path behind the domain you're using.
+
+Mailgun will provide a secret for the webhook. Add this secret to your ``.env``
+file:
+
+.. code-block:: env
+
+    MAILER_MAILGUN_SECRET=your_secret
+
+With this done, you can now add Remote Event Consumers to react to the webhooks::
```

Comment:
```suggestion
With this done, you can now add RemoteEvent Consumer to react to the webhooks::
```
Not sure about space and/or capitalize for consumer

Reply:
a RemoteEvent consumer